examples/Python/SPECT/SPECT_OSEM.py

```diff
@@ -0,0 +1,186 @@
+'''
+Simple OSEM reconstruction demo: creates an image, projects it to simulate
+acquisition data , adds noise to the data and then does an OSEM reconstruction.
+
+Usage:
+  SPECT_OSEM [--help | options]
+
+Options:
+  -f <file>, --file=<file>    raw data file [default: template_sinogram.hs]
+  -p <path>, --path=<path>    path to data files, defaults to data/examples/SPECT
+                              subfolder of SIRF root folder
+  -o <file>, --output=<file>  output file for simulated data
+'''
+## 
+## Author:Sam Porter
```

Comment:
I guess this is very much based on the PET demo, please retain authors

Reply:
I just used acquisition_model.py as a base and then used my own code for the reconstruction stuff (definitely mostly based on the stuff I learned from the summer school notebooks, though). The acquisition_model header didn't have an author.

Not really sure why I put an author at all. I'll just get rid of it

Reply:
Looking back at the notebooks, it's near-identical! I'll remove the author line tomorrow